components/micro-gateway-core/src/main/java/org/ballerina/gateway/siddhi/extension/ThrottleStreamProcessor.java

```diff
@@ -57,7 +57,7 @@
 /**
  * Throttle stream processor class
  */
-@Extension(name = "timeBatch", namespace = "gateway/throttler", description =
+@Extension(name = "timeBatch", namespace = "throttler", description =
```

Comment:
Lets use the old namespace

Reply:
due to an issue in new ballerina this namespace cannot be used